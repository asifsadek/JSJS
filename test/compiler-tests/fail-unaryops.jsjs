val z = ! (/\(x, y) => {
  val zero = 0;
  zero + 5;
});
