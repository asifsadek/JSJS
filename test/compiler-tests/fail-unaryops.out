Type error: expected value of type 'bool', got a value of type '(B, A) -> C' instead
